tests/test_prompt_artifact_risk_path_prompts.py

```
@@ -0,0 +1,34 @@
+        self.assertEqual(len(matches),1); p=matches[0]
+        self.assertEqual((p["id"],p["seq"],p["copySheet"]),("P97","97","P97_COPY_SAFE"))
+        self.assertEqual(p["class"],"ANALYSIS / ARTIFACT RISK TRIAGE")
+        for m in ("OBSERVED","INFERRED","UNKNOWN","PASS 2 — CROSS-ARTIFACT / PROVENANCE REVIEW","DO NOT JUMP STRAIGHT INTO REPAIR","P91 Failure-Class Generalization & Repository Audit","P92 Production-Path Proof Gap Auditor","P93 Use-Case Closure Certification","P94 Regression Test & Live Behavior Guard","If no material risk is supported"):
+            self.assertIn(m,p["copyContent"])
+        self.assertIn("real defect",self.by_id["P91"]["useWhen"].lower()); self.assertNotIn("real defect",p["useWhen"].lower())
+    def test_raw_and_generated_parity(self):
+        spec=json.loads((ROOT/"registry/prompts/spec-architecture-prompts.v1.json").read_text(encoding="utf-8"))
+        raw=next(p for p in spec["prompts"] if p["id"]=="P97")
+        self.assertGreaterEqual(len(raw["copyContent"]),3000); self.assertLessEqual(len(raw["copyContent"]),8000)
+        site=(ROOT/"web/prompt-kit/index.html").read_text(encoding="utf-8")
+        self.assertIn("Artifact Risk Review",site); self.assertIn("MACHINE / OS / SHELL GATE",site)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)